src/tiled/document.cpp

```diff
@@ -112,5 +113,6 @@ void Document::setIgnoreBrokenLinks(bool ignoreBrokenLinks)
     emit ignoreBrokenLinksChanged(ignoreBrokenLinks);
 }
 
+
```

Comment:
No need for adding a newline here.

Reply:
Done

Apologies, slip of the keyboard.